vulnerability-disclosure-policy.md

```diff
@@ -67,3 +67,27 @@ Reports should include:
 Please keep your vulnerability reports current by sending us any new information as it becomes available.
 
 We may share your vulnerability reports with [US-CERT](https://www.us-cert.gov/ais), as well as any affected vendors or open source projects.
+
+### Public Disclosure
+
+This public disclosure policy applies to all submissions under the TTS Vulnerability Disclosure Policy, including Duplicates, Out of Scope, and Not Applicable submissions.
+
+#### Nondisclosure
+
+For 90 days, TTS's default submission disclosure state is nondisclosure for 90 days after submission. Within this 90 day time period, no submissions may be publicly disclosed in full or part without explicit permission from TTS.
```

Comment:
"Default submission disclosure state" doesn't strike me as plain language.

Not a fan of the power dynamic expressed here.  If researchers have found something of value, we should ensure cooperating with us for credit is more attractive than embarrassing us anonymously.

Reply:
Yeah, this is what is already in the policy; I just spelled it out more, but welcome edits.

Reply:
Cool, edits forthcoming.